src/lang/std/engineConnection.ts

```diff
@@ -1194,20 +1201,22 @@ export class EngineCommandManager {
         // this fact is very opaque in the api and docs (as to what should can be deleted).
         // Using an array is the list is likely to grow.
         'start_path',
+        'entity_linear_pattern',
```

Comment:
not everything in the artifact needs to be clean up, because most things are children, in fact start_path was the only thing, since everything was a child of start_path.
But were adding our second now.